blaze/expr/tests/test_collections.py

```diff
@@ -108,13 +108,21 @@ def test_join_option_types():
     assert join(b, a, 'x').dshape == dshape('var * {x: int}')
 
 
-def test_join_mismatched_schema():
+def test_join_exceptions():
+    'mismatched schema; no shared fields'
```

Comment:
> How do you feel about testing the contents of the error strings?

I feel good about it - I had added but wasn't sure if it was overkill. It's easy to add so I'll update.

> Can we use triple quoted strings for docstrings

Yes, will fix.